app/lego.py

```diff
@@ -248,6 +273,9 @@ def startLego(self):
                             previous_tag = current_tag
                         current_tag = identifier
                         # A tag has been matched
+                        if ('playlist' in tags['identifier'][identifier]):
+                            playlist = tags['identifier'][identifier]['playlist']
+                            self.playPlaylist(playlist, mp3_dir)
```

Comment:
Replace line 278 with these to allow optional shuffle playback:
```
if ('shuffle' in tags['identifier'][identifier]):
    shuffle = true
else:
    shuffle = false
self.playPlaylist(playlist, mp3_dir, shuffle)
```

Reply:
ok done